content/03.chemoproteomics.md

```diff
@@ -1,3 +1,44 @@
+Notably, two hyperreactive lysine residues were discovered at the protein interface in co-crystal structures of the tyrosine kinase-like orphan receptor 2 and endoglin, exemplifying the functional relevance of surface-exposed lysine residues.
+[@DOI:10.1038/s41467-024-47033-w]
+
+To selectively probe cysteine residues in cell membrane proteins, the Cys-Surf platform utilizes broad specifity probe with a two-step enrichment strategy.
+After labeling, glycosylated cell surface proteins are enriched using cell surface capture.
+Subequently, proteins are proteolytically digested and probe-bound peptides are enriched by clicking a biotin handle to the probe.
+At this stage, previously unlabeled cysteines—either due to low native reactivity or oxidative modification—can be labeled using an isotopically tagged probe under reducing conditions, allowing quantification of redox states through isotopic ratios.
+Cys-Surf has demonstrated broad utility by recovering targets of FDA-approved covalent drugs, uncovering novel ligandable cysteines at the cell surface and detecting redox changes during T-cell activation. [@DOI:10.1021/jacsau.3c00707]
+
+To investigate non-covalent interactions, crosslinking approaches can be applied.
+Ligands are chemically modified to contain an enrichment handle and a chemically reactive probe which covalently attaches the drug and affinity handle to the interacting proteins.
+Diazirine moieties are widely used for photoaffinity labeling in live-cell studies due to their small size, metabolic stability, and ability to form highly reactive intermediates.
+Inert under ambient light, diazirines generate a short-lived carbene species upon irradiation with long-wavelength UV light, which covalently reacts with molecules within a few angstroms.
+Leveraging this chemistry, researchers have synthesized cholesterol-based probes—and more recently, a 20(S)-hydroxycholesterol analog—equipped with a diazirine probe and alkyne handle to enable spatially confined crosslinking and subsequent enrichment via click chemistry.
+This strategy is particularly powerful due to the inherent challenge of studying interactions of molecules partitioning into membranes using conventional biochemical methods.
+Using this approach, several novel interactions were uncovered, including the stereoselective binding to transmembrane protein 97 (TMEM97).
+The ability to resolve such selective interactions within the complex lipid environment of the membrane highlights the potential of photoaffinity labeling to uncover biologically relevant targets inside of cellular membranes.
+[@DOI:10.1038/s41589-021-00907-2; @DOI:10.1038/nmeth.2368]
+
+Old source list:
+[@DOI:10.1101/2024.08.20.608660; @DOI:10.1038/s41467-024-47033-w; @DOI:10.1021/acschembio.2c00648; @DOI:10.1021/acschembio.3c00637; @DOI:10.1021/jacsau.3c00707; @DOI:10.1038/s41589-021-00907-2]
+
+If you think it is relevent (might be a bit old):
```

Comment:
I think this could be relevant but I would not include it and rather use the limited word count to focus on newer results.